native/src/modules/layout/__tests__/HeaderContainer.spec.js

```diff
@@ -102,4 +105,118 @@ describe('HeaderContainer', () => {
       expect.objectContaining({ shareUrl: url('abc') })
     )
   })
+
+  it('shareUrl should be set correctly for events route', () => {
+    jest.doMock('../components/Header', () => Header)
+    const HeaderContainer = require('../containers/HeaderContainer').default
+
+    const eventsRouteMapping = {
+      routeKeyEvent1: {
+        status: 'ready',
+        path: null,
+        language: language._code,
+        city: city.name,
+        models: [],
+        allAvailableLanguages: new Map()
+      }
+    }
+
+    const state: StateType = prepareState(eventsRouteMapping, {}, {})
+    const store = mockStore(state)
+
+    const ownProps = {
+      scene: {
+        route: {
+          name: EVENTS_ROUTE,
+          key: 'routeKeyEvent1'
+        }
+      }
+    }
+
+    const result = TestRenderer.create(
```

Comment:
Is there a relatively easy way to test the shareUrl via the react native testing library instead of the TestRenderer? Asserting on props is not really good style. Probably quite difficult in this case (or rather: for containers in general).

Reply:
Yeah i tried with react testing library a bit, but wasn't successful. If you have a better idea, how to do this, let me know.

Reply:
Not really. @maxammann any ideas?